ietf-layer0-types-ext.yang

```diff
@@ -51,4 +51,397 @@ module ietf-layer0-types-ext {
+         per second in a digitally 
+         modulated signal or a line code";
+    }
+    leaf available-FEC-type {
+      type identityref {
+      base FEC;
+      }
+      config false;
+      description "Available FEC";
+    }
+    leaf FEC-code-rate {
+      type decimal64 {
+      fraction-digits 8;
+      range "0..max";
+      }
+      config false;
+      description "FEC-code-rate";
+    }
+    leaf FEC-threshold {
+      type decimal64 {
+      fraction-digits 8;
+      range "0..max";
+      }
+      config false;
+    description
+        "Threshold on the BER, for which FEC 
+         is able to correct errors";
+    }
+  } // end grouping of attributes capabilities related to operational mode    
+
+  grouping common-tranceiver-capabilities-organizational-explicit {
+    description "Common capability in case of operational mode 
+    and explicit mode. 
+    These attributes are supported separately in 
+    case of application codes";
+
+/* transmitter tuning range (f_tx-min, f_tx-max) */
+
+    leaf min-central-frequency {
+      type layer0-types:frequency-thz;
+      config false;
+      description
+        "This parameter indicates the minimum frequency for
+         this transceiver.";
+    }
+    leaf max-central-frequency {
+      type layer0-types:frequency-thz;
+      config false;
+      description
+        "This parameter indicates the maximum frequency
+         for this transceiver.";
+    }
+
+/* transmitter-tunability-grid */
+
+    leaf minimum-channel-spacing {
+      type frequency-ghz;
+      config false;
+      description
+        "This paramenter represents the minumum difference in frequency
+        between two adjacent channels (ref. G.698.2 sec.7.1.1).
+        This free value is to permit OTSi's central frequency not to
+        stay on the G.694.1 grid.";
+    } 
+
+/* supported transmitter power range [p_tx-min, p_tx_max] */
+
+    leaf min-channel-output-power {
+      type dbm-t;
+      config false;
+      description "The minimum output power of this interface";
+    }
+    leaf max-channel-output-power {
+      type dbm-t;
+      config false;
+      description "The maximum output power of this interface";
+    } 
+
+/* supported receiver power range [p_rx-min, p_rx_max] */ 
+
+     leaf min-channel-input-power {
+      type dbm-t;
+      config false;
+      description "The minimum input power of this interface";
+    }
+    leaf max-channel-input-power {
+      type dbm-t;
+      config false;
+      description "The maximum input power of this interface";
+    }
+
+    leaf max-total-rx-optical-power {
+      type dbm-t;
+      config false;
+      description "Maximum rx optical power";
+    }      
+
+  } // end grouping of attributes capabilities related to operational mode and explicit mode
+
+/* This grouping represent the list of configured parameters */
+/* values independent of operational mode */
+
+  grouping common-tranceiver-configured-param {

```

Comment:
typo: should be common-transceiver-configured-param

Reply:
done